File: php/Shirt.php
<?php
require_once "Clothing.php";

class Shirt extends Clothing
{

    private $color;
    private $sleeve_type;

    public function __construct()
    {
    }

    public function getColor(): string
    {
        return $this->color;
    }

    public function setColor($color)
    {
        $this->color = $color;
    }

    public function getSleeveT(): string
    {
        return $this->sleeve_type;
    }

    public function setSleeveT($type)
    {
        $this->sleeve_type = $type;
    }
}

File: python/Shirt.py
from Clothing import Clothing


class Shirt(Clothing):
    __color = ""
    __sleeve_type = ""

    def __init__(self, color, sleeve_type):
        self.__color = color
        self.__sleeve_type = sleeve_type

    def getColor(self):
        return self.__color

    def setColor(self, color):
        self.__color = color

    def getSleeveT(self):
        return self.__sleeve_type

    def setSleeveT(self, sleeve_type):
        self.__sleeve_type = sleeve_type
